src/mocks/data.ts

```diff
@@ -3,47 +3,74 @@ import { CartItem } from "~/models/CartItem";
 import { Order } from "~/models/Order";
 import { AvailableProduct, Product } from "~/models/Product";
 
-export const products: Product[] = [
-  {
-    description: "Short Product Description1",
-    id: "7567ec4b-b10c-48c5-9345-fc73c48a80aa",
-    price: 24,
-    title: "Product 1",
-  },
-  {
-    description: "Short Product Description7",
-    id: "7567ec4b-b10c-48c5-9345-fc73c48a80a1",
-    price: 15,
-    title: "Product 2",
-  },
-  {
-    description: "Short Product Description2",
-    id: "7567ec4b-b10c-48c5-9345-fc73c48a80a3",
-    price: 23,
-    title: "Product 3",
-  },
-  {
-    description: "Short Product Description4",
-    id: "7567ec4b-b10c-48c5-9345-fc73348a80a1",
-    price: 15,
-    title: "Product 4",
-  },
-  {
-    description: "Short Product Descriptio1",
-    id: "7567ec4b-b10c-48c5-9445-fc73c48a80a2",
-    price: 23,
-    title: "Product 5",
-  },
-  {
-    description: "Short Product Description7",
-    id: "7567ec4b-b10c-45c5-9345-fc73c48a80a1",
-    price: 15,
-    title: "Product 6",
-  },
-];
+// export const products: Product[] = [
+//   {
+//     description: "Short Product Description1",
+//     id: "7567ec4b-b10c-48c5-9345-fc73c48a80aa",
+//     price: 24,
+//     title: "Product 1",
+//   },
+//   {
+//     description: "Short Product Description7",
+//     id: "7567ec4b-b10c-48c5-9345-fc73c48a80a1",
+//     price: 15,
+//     title: "Product 2",
+//   },
+//   {
+//     description: "Short Product Description2",
+//     id: "7567ec4b-b10c-48c5-9345-fc73c48a80a3",
+//     price: 23,
+//     title: "Product 3",
+//   },
+//   {
+//     description: "Short Product Description4",
+//     id: "7567ec4b-b10c-48c5-9345-fc73348a80a1",
+//     price: 15,
+//     title: "Product 4",
+//   },
+//   {
+//     description: "Short Product Descriptio1",
+//     id: "7567ec4b-b10c-48c5-9445-fc73c48a80a2",
+//     price: 23,
+//     title: "Product 5",
+//   },
+//   {
+//     description: "Short Product Description7",
+//     id: "7567ec4b-b10c-45c5-9345-fc73c48a80a1",
+//     price: 15,
+//     title: "Product 6",
+//   },
+// ];
+
+async function fetchProducts() {
```

Comment:
You don't need to add a fetchProducts method here. The code is pre-written in a way that you only have to add your backend (zq3zdypgs7) to apiPaths.ts. products.ts should then pick up on the path and initiate an Axios request.

Note that you may need to change the /product path to /products in the relevant products.ts methods. I recommend checking the console to see which endpoints are called.

Reply:
I've made the necessary changes in products.ts and apiPaths.ts files to fetch data from corresponding endpoints.